spec/mysql2/client_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'spec_helper'
 
+# rubocop:disable Metrics/BlockLength
```

Comment:
By below modification, the length of the block is more than the limitation.
As I was not sure how to fix, I added this ignored line.

Reply:
See file .rubocop_todo.yml.

But you may be right that the better approach is not to globally increase the limit, but to ignore it for this file and have a more sensible limit apply everywhere else.

Reply:
Okay, I will change current limit 850 to 860 to prevent below situation.

```
spec/mysql2/client_spec.rb:3:1: C: Metrics/BlockLength: Block has too many lines. [851/850]
```